src/components/FeedbackButton.tsx

```diff
@@ -0,0 +1,77 @@
+import { EditIcon } from "@chakra-ui/icons";
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Button,
+    useDisclosure,
+    IconButton,
+    Text,
+    Textarea,
+} from "@chakra-ui/react";
+import React from "react";
+import { submitFeedback } from "../services/feedbackService";
+
+export function FeedbackButton() {
+    const { isOpen, onOpen, onClose } = useDisclosure();
+
+    let [value, setValue] = React.useState("");
+
+    let handleInputChange = (e: any) => {
+        let inputValue = e.target.value;
+        setValue(inputValue);
+    };
+
+    let submitForm = async () => {
+        await submitFeedback(value);
+        alert("Feedback submit!");
```

Comment:
Provide a better Ui maybe for UI submit. Alert is pretty raw. 

You can’t won’t fix this and fix this in a follow up, but we should fix asap